src/app/core/auth/auth.service.ts

```diff
@@ -427,7 +427,14 @@ export class AuthService {
    * Remove authentication token info
    */
   public removeToken() {
-    return this.storage.remove(TOKENITEM);
+    this.storage.remove(TOKENITEM);
+  }
+
+  /**
+   * Check authentication token info
+   */
+  public hasToken(): boolean {
```

Comment:
You could add a simple unit test here which is similar to how `removeToken()` is tested in `auth.service.spec.ts`.  See https://github.com/DSpace/dspace-angular/blob/main/src/app/core/auth/auth.service.spec.ts#L372-L375